addons/skin.confluence/720p/VideoOSD.xml

```diff
@@ -248,9 +254,9 @@
 				<onup>1000</onup>
 				<ondown>1000</ondown>
 				<onclick>PlayerControl(ShowVideoMenu)</onclick>
-				<enable>VideoPlayer.HasMenu</enable>
```

Comment:
@phil65 I also took the liberty to cleanup ```<enable>VideoPlayer.HasMenu</enable>``` here since in line 259 we have the visible condition with the same value.